device/globals/game.gd

```diff
@@ -108,7 +108,19 @@ func clicked(obj, pos):
 			get_tree().call_group_flags(SceneTree.GROUP_CALL_DEFAULT, "hud", "set_tooltip", "")
 
 		elif obj.use_action_menu && action_menu != null:
-			spawn_action_menu(obj)
+			if ProjectSettings.get_setting("escoria/ui/right_mouse_button_action_menu"):
+				if input_event.button_index == 2:
```

Comment:
Better to use `BUTTON_RIGHT` than `2` for clarity.

Reply:
There were other instances of referring to the buttons by numbers; I'll take a moment to see if I can hunt them down.

Reply:
Thanks, that would be awesome. :smile: Not strictly related to this branch of course, but I appreciate the extra effort.